src/components/pages/ServiceMarketplace/RecommendedServices.tsx

```diff
@@ -73,7 +73,11 @@ export default function RecommendedServices({
               <CardHorizontal
                 borderRadius={6}
                 imageAlt="App Card"
-                imagePath={`${getAssetBase()}/images/content/ServiceMarketplace.png`}
+                imagePath={
+                  service?.leadPictureId
+                    ? `${getApiBase()}/api/administration/documents/${service.leadPictureId}`
+                    : `${getAssetBase()}/images/content/ServiceMarketplace.png`
+                }
```

Comment:
This code is used in the below file. Consider converting it into a reusable function for efficiency.